tests/frameworks/layers/l2l4_network_creation_test.py

```diff
@@ -193,6 +193,52 @@ def testL4L2ColumnCreate(self):
     net.run(3)
 
 
+  def testL4L2ColumnLinks(self):
+    """
+    In this simplistic test we create a network and ensure that it has the
+    correct links between regions.
+    """
+
+    # Create a simple network to check its architecture
+    net = createNetwork(networkConfig1)
+
+    links = net.getLinks()
+
+    # Make sure that we have the right number before going on to specifics
+    self.assertEqual(len(list(net.getLinks())), 6, "Incorrect number of links")
+
+    # These are all the links we're hoping to find
+    desired_links=[("sensorInput_0", "dataOut", "L4Column_0", "activeColumns"),
```

Comment:
This heavily relies on implementation details for constructing the region "names". Is there an easy way to get the actual name as opposed to the derived one (with "_0")? That would be a better test.

You might also make this a set of tuples and simply check for set equality below. For the debug message you can just print both full lists since there aren't too many links to manually inspect.